dask-gateway-server/setup.py

```diff
@@ -97,6 +97,8 @@ def run(self):
         _clean.run(self)
 
 
+# NOTE: changes to the dependencies here must also be reflected
+# in ../dask-gateway-dev.yaml
 install_requires = ["aiohttp", "colorlog", "cryptography", "traitlets"]
```

Comment:
Looking through this I found a `import tornado` statement. Since we explicitly depend on tornado, we should make it explicit in our install_require.

```suggestion
# NOTE: changes to the dependencies here must also be reflected
# in ../dev-environment.yaml
install_requires = ["aiohttp", "colorlog", "cryptography", "tornado", "traitlets"]
```